@ecomplus/widget-search-engine/src/index.js

```diff
@@ -144,7 +143,7 @@ export default (options = {}, elId = 'search-engine', paginationElId = 'search-p
                   vm.countRequests++
                   const renderNewItems = () => {
                     vm.canShowItems = true
-                    $('#search-engine-snap').remove()
+                    document.body.removeChild(document.getElementById('search-engine-snap'))
```

Comment:
Isso aqui não vai funcionar só se o elemento for filho do `body` diretamente?
Não é melhor usar https://developer.mozilla.org/en-US/docs/Web/API/Element/remove ?

Reply:
Só não funciona no IE https://developer.mozilla.org/en-US/docs/Web/API/Element/remove#browser_compatibility
Mas tem muita coisa no storefront que não funciona no IE 😛